odoo/addons/base/ir/ir_autovacuum.py

```diff
@@ -32,6 +32,8 @@ def _gc_user_logs(self):
 
     @api.model
     def power_on(self):
+        if not self.env.user._is_superuser():
+            raise UserError(_("Only the admin user is allowed to execute this action."))
```

Comment:
@fmdl UserError and _ are not imported

Reply:
There is also no point raising a UserError with a translated error message for something that is not supposed to happen ;-)